book/opmanual_autolab/15_autolab_minimal_requirements/40_autolab_map_editor.md

```diff
@@ -0,0 +1,70 @@
+# Autolab map {#autolab-map-editor status=draft}
+
+
+<div class='requirements' markdown="1">
+
+Requires: Knowledge about what a Autolab is [](#part:autolab-definition)
+
+Results: generated map for duckietown
+</div>
+
+## Map editor
+
+You can launch map editor, using command:
+
+    laptop $ dts map editor
+
+You will see:
+
+<div figure-id="fig:editor1" figure-caption="Map editor">
+  <img src="images/simple_editor.png" style='width: 25em; height:auto'/>
+</div>
+
+## How to create new map?
+
+You should specify size(width, height) of map and size of tile. Default tile size: **0.585**
+<div figure-id="fig:editor2" figure-caption="How to create new map">
+  <img src="images/create_map.png" style='width: 25em; height:auto'/>
+</div>
+
+Result of generation map
```

Comment:
`map generation` or `generation of the map`?

Reply:
```suggestion
Then the map generation process should be initialized. Sample editor window:
```